packages/web-components/fast-foundation/src/menu/menu.spec.md

```diff
@@ -109,15 +83,14 @@ The third item in the menu has a glyph, which would inset the text from the othe
 
 *Slots:*
 - before - slot which precedes content
-- content - slot for the content (the default slot for the item)
+- default - slot for the content (the default slot for the item)
 - after - slot which comes after content
-- menu - the slot used to generate a submenu
+- submenu - the slot used to generate a submenu
 
 *CSS Parts*
 - before
 - content
 - after
-- menu
 
```

Comment:
Shouldn't there be a `submenu` part?

Reply:
added "submenu-region" for the layout area.